### PR TITLE
AG-1307 - Flatten ensembl_info

### PR DESCRIPTION
**Pull request by sagely1**
Flattening ensembl_info to match data

### CODE REVIEW

File: src/server/models/genes.ts [hunk 1]
```diff
@@ -77,7 +77,7 @@ const GeneSchema = new Schema<Gene>(
     median_expression: { type: [MedianExpressionSchema], required: true },
     druggability: { type: [DruggabilitySchema], required: true },
     total_nominations: { type: Number, required: true },
-    ensembl_info: { type: [EnsemblInfoSchema], required: true }
+    ensembl_info: { type: EnsemblInfoSchema, required: true }
```
Comment (sagely1) [inline src/server/models/genes.ts:80]: no longer an array in the data

File: src/app/models/genes.ts [hunk 2]
```diff
@@ -101,8 +101,7 @@ export interface Gene {
 
   bio_domains?: BioDomains;
 
-  // FIXME Will be fixed by AG-1324 in a future data release as it doesn't make sense to have an array here
```
Comment (sagely1) [inline src/app/models/genes.ts:104]: now we can remove the FIXME

File: src/app/features/genes/components/gene-hero/gene-hero.component.ts [hunk 3]
```diff
@@ -103,9 +103,9 @@ export class GeneHeroComponent {
   }
 
   getEnsemblUrl() {
-    if (!this.gene?.ensembl_info || this.gene.ensembl_info.length <= 0)
```
Comment (hallieswan) [inline src/app/features/genes/components/gene-hero/gene-hero.component.ts:106]: Could consider adding a test to confirm that an empty string will be returned when `ensembl_info` is falsy (similar to the test for when `ensembl_info` was an empty array, which was understandably removed in this PR), but up to you!

**Review by hallieswan — APPROVED** [review]
Looks good to me!
